hazelcast/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -613,6 +619,29 @@ public ClientConfig setLoadBalancer(LoadBalancer loadBalancer) {
         return this;
     }
 
+    /**
+     * Gets load balancer class name
+     *
+     * @return load balancer class name
+     * @see com.hazelcast.client.LoadBalancer
+     */
+    public String getLoadBalancerClassName() {
+        return loadBalancerClassName;
+    }
+
+    /**
+     * Sets load balancer class name
+     *
+     * @param loadBalancerClassName {@link LoadBalancer}
+     * @return configured {@link com.hazelcast.client.config.ClientConfig} for chaining
+     * @see com.hazelcast.client.LoadBalancer
+     */
+    public ClientConfig setLoadBalancerClassName(String loadBalancerClassName) {
+        this.loadBalancerClassName = loadBalancerClassName;
+        return this;
+    }
```

Comment:
Can you make the implementation aligned with ListenerConfig. 
If loadBalancer is set it will be set to null. Also the class name will be set to null when the loadBalacer is set. 
Please do not forget to add related javadoc. 

And also as it is implemented in ListenerConfig add @Nonnull  tag together with `checkHasText`

Reply:
Done. I made them mutually exclusive as suggested.